docs/contracts/NFT Contract/NFT Actions/purchase.a.md

```diff
@@ -74,6 +74,8 @@ Internally we are constructing a vector of which `token_ids` to be transferred.
 | receiver         | eosio::asset                        | string          | Account that will receive the Uniq from this purchase                                                                                                                               |
 | promoter_id      | std::optional\<eosio::name>         | string / null   | Optional promoter of the purchase transaction. If no promoter is provided then the default promoter specified in `saleshrlmcfg` (scope `0`) will be used if present                 |
 | user_uniqs       | std::optional\<provided_user_uniqs> | Array / null    | List of uniqs the buyer is willing to provide for this purchase option to either be taken from him or to just verify their presence. Refer to `provided_user_uniqs` breakdown below |
+| start            | std::optional\<time_point_sec>      | string / null          | Start time of purchase window (optional)
+| end            | std::optional\<time_point_sec>      | string / null          | End time of purchase window (optional)
```

Comment:
The names used for the purchase period fields are not very clear. Better use purchase_window_start and purchase_window_end instead to clarify the purpose.

Also the formatting on the table is a bit broken. Ideally you should use some extension like this: https://marketplace.visualstudio.com/items?itemName=TakumiI.markdowntable

Reply:
not sure why you say formatting is broken. looks good to me

https://github.com/ultraio/docs-blockchain/blob/feature/BLOCK-1775-purchase-window/docs/contracts/NFT%20Contract/NFT%20Actions/purchase.a.md

Reply:
I mean not in the rendered doc, but the raw markdown view. The extension I linked adjusts the spacing in the markdown so it also looks like a normal table when looking at it even in the raw view